libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/list/info/InfoListItemDiffCallback.kt

```diff
@@ -0,0 +1,19 @@
+package com.trendyol.uicomponents.dialogs.list.info
+
+import androidx.recyclerview.widget.DiffUtil
+
+class InfoListItemDiffCallback : DiffUtil.ItemCallback<Pair<CharSequence, CharSequence>>() {
+    override fun areContentsTheSame(
+        oldItem: Pair<CharSequence, CharSequence>,
+        newItem: Pair<CharSequence, CharSequence>
+    ): Boolean {
+        return oldItem.first == newItem.first && oldItem.second.toString() == newItem.second.toString()
```

Comment:
Is there any specific reason to check `first` as CharSequence and check `second` as `String`? If no, both `areContentsTheSame` and `areItemsTheSame` can be checked with `oldItem == newItem`.

Reply:
You are right. 
* fixed